Parent/Parent/Routes.swift

```diff
@@ -44,6 +44,10 @@ let router = Router(routes: [
     RouteHandler(.courseGrades(":courseID")) { _, params in
         guard let courseID = params["courseID"] else { return nil }
         guard let studentID = currentStudentID else { return nil }
+        guard ExperimentalFeature.parent3.isEnabled else {
+            guard let session = legacySession else { return nil }
+            return CalendarEventWeekPageViewController.create(session: session, studentID: studentID, courseID: courseID)
+        }
```

Comment:
This was added to see progress of grades.  We now have another way to get to grades currently.  Do we need this route on parent?

Reply:
Or is the route you want used for new grades stuff?

Reply:
I think we want this route _eventually_ so that if a teacher links to the grades page from somewhere like the syllabus we show them the native view.

Do you think we should remove the route entirely for now? I am totally open to that. Having it makes it nice to work on Parent 3 though..

Reply:
Sounds great 👍 , let's keep it